rpg-1e-stats/Makefile

```diff
@@ -1,27 +1,30 @@
-#TARGET=test_gnoll.pdf
 TARGET=test_all.pdf
 
 all: $(TARGET)
 
 show: $(TARGET)
 	evince $(TARGET)
 
+$(TARGET): rpg-1e-stats.def
+
 %.pdf: %.tex
 	pdflatex $<
 
-convert:
-	./convert
+%.def: %.csv convert
+	./convert $<
 
-rpg-1e-stats.def: convert
+rpg-1e-stats.csv: mm.csv ff.csv mm2.csv
```

Comment:
rpg-1e-stats.csv depends upon these three files and how to put them together to do it!